src/Fields/constant_field.jl

```diff
@@ -9,6 +9,12 @@ OneField(T=Int) = OneField{T, 3}() # default 3D, integer 1
 @inline Base.getindex(::ZeroField{T, N}, ind...) where {N, T} = zero(T)
 @inline Base.getindex(::OneField{T, N}, ind...) where {N, T} = one(T)
 
+@inline Base.summary(::ZeroField{T}) where T = string("ZeroField with type $T")
+@inline Base.summary(::OneField{T})  where T = string("OneField with type $T")
+
+Base.show(io::IO, f::ZeroField{T}) where T = print(io, summary(f))
+Base.show(io::IO,  f::OneField{T}) where T = print(io, summary(f))
```

Comment:
why these? what about https://github.com/CliMA/Oceananigans.jl/blob/86d96cadda4af724bcc9141c545d5ad55806a398/src/Fields/show_fields.jl#L60

Reply:
I'm dropping them. I'll keep the show for `OneField` but move to `show_fields.jl` file

Reply:
Also, btw, let me remark that 

```julia
Base.show(io::IO, f::ZeroField{T}) where T = print(io, summary(f))
Base.show(io::IO,  f::OneField{T}) where T = print(io, summary(f))
```
are the same function, so it's duplicate code. This is better:

```julia
Base.show(io::IO, f::Union{ZeroField, OneField}) = print(io, summary(f))
```